builds/misc/templates/mqtt-perf-run.yaml

```diff
@@ -0,0 +1,265 @@
+jobs:
+
+  ################################################################################	
+  # Fan-in Test cases (many publishers/few subscribers)
+  ################################################################################	
+  
+  ### QoS 0
+  
+  - template: mqtt-perf-test-case.yaml
```

Comment:
what is test duration for each test? I am worried that it seems current implementation is not easy to scale out.  we should make it possible to run on different pair of pub/sub agents, therefore we can add more agents if we want to shorten the total time for running all MQTT perf test.

Reply:
how about use job strategy (https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#define-a-single-job)?

Reply:
The default test duration is 2m. It is parametrizable. 

These all good points. I'm not sure if we are going to invest into making tests more scalable at this moment. Our goal was to get some initial perf numbers, compare to mosquitto and track them going forward to catch regressions. //cc @varunpuranik if he has any thoughts on this.

Reply:
is 2 min too short for a performance test?

Yes. confirm with @varunpuranik to make sure we don't over engineering at this point of time. 👍

Reply:
We decided that 2min is fine, taking into account that we have a lot of test cases to cover. These are perf tests, and not stress tests, and 2m is fine for now to establish baseline and compare with mosquitto.

Reply:
I think perf test is to check speed/response time and scalability of a software under their expected workload. If expected workload can be test in 2 mins, that's fine. :)